example/android/app/src/main/AndroidManifest.xml

```diff
@@ -9,7 +9,8 @@
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
-        android:theme="@style/AppTheme">
+        android:theme="@style/AppTheme"
+        android:usesCleartextTraffic="true">
```

Comment:
Why do we need clear text traffic?

Reply:
@akshaybhalotia while connect to react native debugger its use for that .

Reply:
Let's remove it from here, if some devs need it they should be able to add it themselves. Running the debugger is not the primary purpose of the sample app, it is to show an ideal implementation in which case clear text shouldn't be allowed.